components/gc-follow-us/_base.scss

```diff
@@ -182,6 +182,10 @@ $social-media-icons-size: 38px;
 			@extend %social-media-icons-gc-followus-logo-properties;
 			background-image: url("../assets/gc-follow-us/linkedin.svg");
 		}
+
+		.more-ways {
+			display: block;
+		}
```

Comment:
This breaks the styling for the link
![image](https://github.com/user-attachments/assets/0921824b-1c56-4674-a543-8e8b5c675d2d)

Reply:
You must have put it at the wrong spot because it works for me. Make sure it is outside the "li:not(.more-ways)" definition, but inside the "&.list-inline" definition.